exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -1095,7 +1104,7 @@ public GroupScan applyFilter(LogicalExpression filterExpr, UdfUtilities udfUtili
 
     final Set<SchemaPath> schemaPathsInExpr = filterExpr.accept(new ParquetRGFilterEvaluator.FieldReferenceFinder(), null);
 
-    final List<RowGroupMetadata> qualifiedRGs = new ArrayList<>(parquetTableMetadata.getFiles().size());
+    final List<RowGroupInfo> qualifiedRGs = new ArrayList<>(rowGroupInfos.size());
```

Comment:
Never mind the previous comment. It's probably better to use RowGroupInfos throughout the code.